app/models/test/client_request.rb

```diff
@@ -0,0 +1,23 @@
+module Test
```

Comment:
Is there going to be any confusion that a AR model class is not with all the other AR model classes?